src/pages/plan/utils/WaypointDispatcherResolver.ts

```
@@ -0,0 +1,36 @@
+  WayPointInitType,
+  WayPointResponseType,
+  WayPointUpdateType,
+} from '../types/WaypointResponseBodyType';
+
+export function WaypointDispatcherResolver(message: any) {
+  const wpData: WayPointResponseType = JSON.parse(message.body);
+  console.log('WAYPOINT 메시지:', wpData);
+  switch (wpData.type) {
+    case 'INIT':
+      const wpInit = wpData as WayPointInitType;
+      for (let i = 0; i < wpInit.WAYPOINT.length; i++) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)